modules/ROOT/pages/rtm-agent-config-yaml.adoc

```diff
@@ -0,0 +1,145 @@
+= Update Agent Configuration in mule-agent.yml
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: agent, runtime manager, mule, esb, servers, monitor, notifications, external systems, third party, get status, metrics
+
+_Enterprise Edition_
+
+image:logo-cloud-disabled.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
+image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", title="Hybrid Deployment"]
+image:logo-server-active.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
+
+
+When you run `amc_setup` with parameters to install Runtime Manager agent, the agent configuration is written to the `$MULE_HOME/conf/mule-agent.yml` file. At startup, the agent reads its configuration from this file.
+
+You can further configure or change the agent configuration by manually adding or editing parameters in the `mule-agent.yml` file.
+
+You can configure some agent components during runtime. For information, see xref:administration-service.adoc[Administration Service].
+
+
+== mule-agent.yml File Structure
+
+The `mule-agent.yml` file has the following structure:
+
+. Component types
++
+Transports, services, internal handlers, and external handlers
++
+.. Component name
++
+For example, `mule.agent.jmx.publisher.service`
++
+... Component configuration
++
+A component can have complex object configurations, including more than one level of recursion.
+
+Here is an example `mule-agent.yml` file:
+
+[source,yaml,linenums]
+----
+
+muleInstanceUniqueId: validId
```

Comment:
I think this example is wrong. I think the content of a mule-agent.yml is like this (i am taking this from my runtime):
```
transports:
  rest.agent.transport:
    enabled: false
  websocket.transport:
    enabled: true
    consoleUri: wss://mule-manager.anypoint.mulesoft.com:443/mule
    handshake:
      enabled: true
      body: null
globalConfiguration:
  security:
    keyStorePassword: keyStorePass
    keyStoreAlias: agent
    keyStoreAliasPassword: keyStoreAliasPass
  authenticationProxy:
    endpoint: https://arm-auth-proxy.cloudhub.io
internalHandlers:
  mule.agent.tracking.handler.analytics:
    enabled: false
services:
  mule.agent.logging.service:
    applicationLogLevels:
    - artifactName: application-1
      scopeLoggingConfigurations: []
    - artifactName: application-2
      scopeLoggingConfigurations: []
    enabled: true
```

Reply:
Thanks. Replaced.